go.mod

```diff
@@ -6,9 +6,12 @@ replace github.com/red-hat-storage/ocs-operator/api/v4 => ./api
 
 replace github.com/red-hat-storage/ocs-operator/metrics/v4 => ./metrics
 
+replace github.com/ceph/ceph-csi-operator => github.com/nb-ohad/ceph-csi-operator v0.0.0-20240731113052-89481456292a
+
 require (
 	github.com/RHsyseng/operator-utils v1.4.13
 	github.com/blang/semver/v4 v4.0.0
+	github.com/ceph/ceph-csi-operator v0.0.0-00010101000000-000000000000
```

Comment:
I believe there is no need to load the entire `ceph-csi-operator` package as a dependency but take only the API sub-packge

Reply:
moved to api module as it is available now.